browser-extension/mv2/src/client/js/ruleExecutionHandler.js

```diff
@@ -18,7 +18,8 @@ RQ.RuleExecutionHandler.setup = () => {
           message.appliedRuleDetails,
           message.isConsoleLoggerEnabled
         );
-        break;
+        sendResponse(true);
```

Comment:
Do you need to send `true` as argument?

Reply:
No, I dont need to send `true` but we are using a callback in the sender function so a response is expected. Hence, we were getting the error.